src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -60,11 +81,11 @@ def geographic_circles(self):
             for i in range(self.geo_circles_amount)
         ]
 
-    @cached_property
+    @property
```

Comment:
These are values that won't change.
just make them members in an __init__ function.
Yeah, that's not how it it right now in Consts but it should be